tools/deployment/package-helm/values.yaml

```diff
@@ -15,8 +15,8 @@ securityContext:
 image:
   clpPackage:
     repository: "ghcr.io/y-scope/clp/clp-package"
-    pullPolicy: "IfNotPresent"
-    tag: "nightly"
+    pullPolicy: "Always"
+    tag: "main"
```

Comment:
these should be changed in a separate PR

@coderabbitai create an issue to track